patch.go

```diff
@@ -16,6 +16,11 @@ import (
 // At the end of the test (see "Deferred execution" in the package docs), the
 // destination is set back to its original value.
 func (c *C) Patch(dest, value interface{}) {
+	cleanup := patch(dest, value)
+	c.cleanup(cleanup)
+}
+
+func patch(dest, value interface{}) func() {
```

Comment:
Rather than factoring out this function, maybe we could just define the top level Patch function in terms of C.Patch?

```
func Patch(t testing.TB, dest, value interface{}) {
    qt.New(t).Patch(dest, value)
}
```

same with the other things. Marginally simpler and easier to understand, arguably.

Reply:
Done.